src/Core/tests/DeviceTests/Handlers/DatePicker/DatePickerHandlerTests.Windows.cs

```diff
@@ -60,6 +60,45 @@ public async Task FormatInitializesCorrectly(string format, string nativeFormat)
 			await ValidatePropertyInitValue(datePicker, () => datePicker.Format, GetNativeFormat, format, nativeFormat);
 		}
 
+		[Theory(DisplayName = "Standard Format Strings Initialize Correctly")]
```

Comment:
When testing with DeviceTest , I facing issue on MaximumDate , 'The UTC time represented when the offset is applied must be between year 0 and 10,000. (Parameter 'offset')'